File: app/src/main/java/com/borisruzanov/russianwives/mvp/presenter/SliderFragmentsPresenter.java
package com.borisruzanov.russianwives.mvp.presenter;

import android.net.Uri;

import com.arellomobile.mvp.MvpPresenter;
import com.arellomobile.mvp.MvpView;
import com.borisruzanov.russianwives.mvp.model.interactor.SliderInteractor;
import com.borisruzanov.russianwives.ui.slider.SliderImageFragment;

public class SliderFragmentsPresenter extends MvpPresenter<MvpView> {

    SliderInteractor interactor;
    SliderImageFragment sliderImageFragment;

    public SliderFragmentsPresenter(SliderInteractor interactor, SliderImageFragment sliderImageFragment) {
        this.interactor = interactor;
        this.sliderImageFragment = sliderImageFragment;
    }



}

File: app/src/main/java/com/borisruzanov/russianwives/mvp/ui/slider/SliderFragmentsPresenter.kt
package com.borisruzanov.russianwives.mvp.ui.slider

import com.arellomobile.mvp.MvpPresenter
import com.arellomobile.mvp.MvpView
import com.borisruzanov.russianwives.mvp.model.interactor.slider.SliderInteractor

class SliderFragmentsPresenter(var interactor: SliderInteractor)
    : MvpPresenter<MvpView>()
